test/cli_rich_container_test.go

```diff
@@ -259,3 +321,42 @@ func (suite *PouchRichContainerSuite) TestRichContainerUpdateEnvFile(c *check.C)
 		c.Errorf("failed to update env, got content: %s", output)
 	}
 }
+
+// TestRunRichModeContainerWithoutRich tests running container with --rich-mode without --rich.
+func (suite *PouchRichContainerSuite) TestRunRichModeContainerWithoutRich(c *check.C) {
+	name := "TestRunRichModeContainerWithoutRich"
+
+	res := command.PouchRun("run", "-d", "-m", "300M",
```

Comment:
a little question here: 300m is required ?

Reply:
not necessary, but it's ok to specify memory when create a container, WDYT?